<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Outer_after_widget</class>
 <widget class="QWidget" name="Outer_after_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#outer_after
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>199</y>
     <width>1185</width>
     <height>483</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#scrollArea
{
border:1px solid black;
border-top:none;

}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1183</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>169</y>
     <width>1185</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#widget
{
border:1px solid black;

}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QPushButton
{
font-family: '黑体';
}</string>
   </property>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>5</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加部件进行计算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fold">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>2</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>全部折叠/展开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>4</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_info" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1185</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_info
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主竖区:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>甲板: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>室外编号: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>噪声限值: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>噪音源支管数量:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MVZName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_deck">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_room_number">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_noise_limit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_duct_num">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
